packages/components/scripts/generateThemesObject.ts

```diff
@@ -7,6 +7,10 @@ import tinycolor2 from 'tinycolor2';
 import { pascalCase } from 'change-case';
 
 const darkenColor = (color: string) => tinycolor2(color).darken(12).toHexString();
```

Comment:
FYI:

'darkenColor' is being used for 'hover background' for light theme, no colorful (i.e. backgroundColor or contrastHighColor)
'darkenColorSlightly' is being used for 'hover backround' for light theme, colorful (i.e. success, error, warning, information)

'lightenColor' - the same as 'darkenColor', only for dark theme
'lightenColorSlightly' - the same as 'darkenColorSlightly', only for dark theme